pom.xml

```diff
@@ -301,17 +308,19 @@
       <version>${jetty.version}</version>
     </dependency>
     -->
-
+    <!--
+    spdy abandonned
```

Comment:
Nope I don't think so.
FYI I have some test locally to enable http2 as well. But not fully test so will do an other pr later.